benchmarks/summary.md

```diff
@@ -22,6 +22,8 @@ Date | Device  | dtype | batch size | cache length |max input length |max output
 ----| ------- | ------ |---------- | -------------|-----------------|------------------|----------------------
 2024-05-14 | TPU v5e-8 | bfloat16 | 512 | 2048 | 1024 | 1024 | 8700
 2024-05-14 | TPU v5e-8 | int8 | 1024 | 2048 | 1024 | 1024 | 8746
+2024-06-13 | TPU v5e-1 | bfloat16 | 1024 | 2048 | 1024 | 1024 | 4249
```

Comment:
The v5e-1 number looks great! The gap is huge, I'm wondering the v5-8 data parallel didn't compute each token locally.

Reply:
+1. Since we are sharding on batch for Gemma 2B, even at naive case (duplicate weights) v5e-8 should have 8 * 4249 = 33k toks/sec

Reply:
The prefills are replicated (8 chips computing the same thing) because we are sharding on batch and prefill is of batch=1. There might be other issues. Overall we should recommand v5e-1 for gemma-2b instead.